### PR TITLE
Pullrequest callkit aes encrypted caller

### PR DESCRIPTION
**Pull request by BenSartor**

### CODE REVIEW

File: Simlar/src/SMLRAesUtil.m [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                                                kCCAlgorithmAES128,
+                                                kCCOptionPKCS7Padding,
+                                                [keyData bytes], kCCKeySizeAES256,
+                                                [ivData bytes],
+                                                [encryptedData bytes], encryptedDataSize,
+                                                output, outputSize,
+                                                &decryptedBytes);
+
+    if (cryptStatus == kCCSuccess) {
+        return [[NSString alloc] initWithData:[NSData dataWithBytesNoCopy:output
```
[the remark this answers is not in the document]
Comment (BenSartor) [reply Simlar/src/SMLRAesUtil.m]: Hi @jritzerfeld,
I have just pushed a commit with that. And I'm totally fine with it.

[the remark this answers is not in the document]
Comment (BenSartor) [reply Simlar/src/SMLRAesUtil.m]: But as I know you read the docs:
Does this realy change anything?

https://developer.apple.com/documentation/foundation/nsdata/1547229-datawithbytesnocopy?language=objc

**Comment by BenSartor** [comment]
Hi @jritzerfeld,
there is another new commit: 450fc07
Cheers,
 Ben

File: Simlar/src/SMLRAesUtil.m (hunk 1 above)
[the remark this answers is not in the document]
Comment (jritzerfeld) [reply Simlar/src/SMLRAesUtil.m]: Hehe, sure I do, but it looks like the more specific docs, and some books, are less helpful here: https://developer.apple.com/documentation/foundation/nsdata/1547240-datawithbytesnocopy?language=objc
Your code is absolutely fine.

**Pull request merged**